src/priority_queue/iterators.rs

```diff
@@ -105,6 +111,7 @@ impl<'a, I: 'a, P: 'a, H: 'a> Drop for IterMut<'a, I, P, H>
 where
     I: Hash + Eq,
     P: Ord,
+    H: BuildHasher,
```

Comment:
Is this change needed?

Reply:
I think so.

Needed by `MutableKeys` trait